<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1462</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{background-color:#424242; color:white;}
QPushButton{background-color:black; color:white;}
QTextEdit{color:black;}
QFontComboBox{background-color:black;}
QSpinBox{background-color:black;}
QLineEdit{background-color:#C8C8C8; color:black;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="maximumSize">
      <size>
       <width>116</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_6">
     <property name="maximumSize">
      <size>
       <width>116</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButton_8">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Undo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_9">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Redo</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton_5">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QPushButton" name="pushButton_7">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="10">
    <widget class="QLineEdit" name="lineEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Open a file.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="13">
    <widget class="QTextEdit" name="textEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit#textEdit{background-color:white;}
</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_10">
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Font to Bold</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButton_14">
     <property name="maximumSize">
      <size>
       <width>97</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Normalize font</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_16">
     <property name="maximumSize">
      <size>
       <width>74</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Bold_Italic</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_13">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QPushButton" name="pushButton_15">
     <property name="maximumSize">
      <size>
       <width>121</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Bold_Underline</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButton_11">
     <property name="maximumSize">
      <size>
       <width>83</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Font to Italic</string>
     </property>
    </widget>
   </item>
   <item row="3" column="8">
    <widget class="QPushButton" name="pushButton_2">
     <property name="maximumSize">
      <size>
       <width>116</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Zoom in</string>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QPushButton" name="pushButton_3">
     <property name="maximumSize">
      <size>
       <width>116</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Zoom out</string>
     </property>
    </widget>
   </item>
   <item row="3" column="10">
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>23</height>
      </size>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="11">
    <widget class="QSpinBox" name="spinBox_2">
     <property name="maximumSize">
      <size>
       <width>43</width>
       <height>23</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Font size</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="3" column="12">
    <widget class="QPushButton" name="pushButton_12">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
